2021-12-10 16:36:12.384 [error] <0.304.0> Supervisor 'Elixir.AuctionApplication.Supervisor' had child 'Elixir.AuctionApplication.UserSystem' started with 'Elixir.AuctionApplication.UserSystem':start_link([]) at undefined exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 631 in context start_error
2021-12-10 16:36:12.384 [error] <0.310.0> CRASH REPORT Process <0.310.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 631
2021-12-10 16:36:12.384 [error] <0.302.0> CRASH REPORT Process <0.302.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.AuctionApplication.UserSystem',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>},[{'Elixir.Supervisor',init_child,1,[{file,"lib/supervisor.ex"},{line,631}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1582}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1582}]},{'Elixir.Supervisor',...},...]}}},...} in application_master:init/4 line 142
2021-12-10 16:39:36.074 [error] <0.284.0> Supervisor 'Elixir.AuctionApplication.Supervisor' had child 'Elixir.AuctionApplication.UserSystem' started with 'Elixir.AuctionApplication.UserSystem':start_link([]) at undefined exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 631 in context start_error
2021-12-10 16:39:36.074 [error] <0.290.0> CRASH REPORT Process <0.290.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 631
2021-12-10 16:39:36.074 [error] <0.282.0> CRASH REPORT Process <0.282.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.AuctionApplication.UserSystem',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>},[{'Elixir.Supervisor',init_child,1,[{file,"lib/supervisor.ex"},{line,631}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1582}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1582}]},{'Elixir.Supervisor',...},...]}}},...} in application_master:init/4 line 142
2021-12-10 16:40:11.300 [error] <0.298.0> Supervisor 'Elixir.AuctionApplication.Supervisor' had child 'Elixir.AuctionApplication.UserSystem' started with 'Elixir.AuctionApplication.UserSystem':start_link([]) at undefined exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 631 in context start_error
2021-12-10 16:40:11.300 [error] <0.304.0> CRASH REPORT Process <0.304.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>} in 'Elixir.Supervisor':init_child/1 line 631
2021-12-10 16:40:11.300 [error] <0.296.0> CRASH REPORT Process <0.296.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.AuctionApplication.UserSystem',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module AuctionApplication.UserManager was given as a child to a supervisor but it does not exist.">>},[{'Elixir.Supervisor',init_child,1,[{file,"lib/supervisor.ex"},{line,631}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1582}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1582}]},{'Elixir.Supervisor',...},...]}}},...} in application_master:init/4 line 142
2021-12-10 16:43:57.897 [error] <0.305.0>@Elixir.AuctionApplication.AuctionManager:handle_cast:94 gen_server 'Elixir.AuctionApplication.AuctionManager' terminated with reason: bad arithmetic expression in 'Elixir.AuctionApplication.AuctionManager':handle_cast/2 line 94
2021-12-10 16:43:57.906 [error] <0.305.0>@Elixir.AuctionApplication.AuctionManager:handle_cast:94 CRASH REPORT Process 'Elixir.AuctionApplication.AuctionManager' with 0 neighbours crashed with reason: bad arithmetic expression in 'Elixir.AuctionApplication.AuctionManager':handle_cast/2 line 94
2021-12-10 16:43:57.906 [error] <0.303.0>@Elixir.AuctionApplication.AuctionManager:handle_cast:94 Supervisor 'Elixir.AuctionApplication.AuctionSystem' had child 'Elixir.AuctionApplication.AuctionManager' started with 'Elixir.AuctionApplication.AuctionManager':start_link([]) at <0.305.0> exit with reason bad arithmetic expression in 'Elixir.AuctionApplication.AuctionManager':handle_cast/2 line 94 in context child_terminated
